김효중/handle_css.js

```diff
@@ -0,0 +1,35 @@
+let major = document.querySelector('.Magor');
```

Comment:
```suggestion
const major = document.querySelector('.Magor');
```
JS는 다음 주차에 진행될 내용이지만 몇자 적어볼게요

재할당이 일어나지 않는 경우에는 `const`로 선언하시는 것을 권장해요.